cpp/src/strings/search/find.cu

```diff
@@ -372,6 +373,42 @@ CUDF_KERNEL void contains_warp_parallel_fn(column_device_view const d_strings,
   if (lane_idx == 0) { d_results[str_idx] = result; }
 }
 
+CUDF_KERNEL void multi_contains_warp_parallel_fn(column_device_view const d_strings,
+                                                 cudf::device_span<string_view> d_targets,
+                                                 cudf::device_span<bool*> d_results)
+{
+  auto const num_targets = d_targets.size();
+  auto const num_rows    = d_strings.size();
+
+  auto const idx = static_cast<size_type>(threadIdx.x + blockIdx.x * blockDim.x);
```

Comment:
Every time you access `idx` you are extracting the warp, so why add the thread index?

```
auto const warp_idx = static_cast<size_type>(blockIdx.x * blockDim.x);
...
if (warp_idx > num_rows * num_targets) { return; }

auto const lane_idx = thread_idx.x;
auto const str_idx = warp_idx % num_rows;
auto const target_idx = warp_idx / num_rows;
```

Reply:
That is a good point. I'm not sure the value of this suggestion.